app/ui/legacy/src/main/java/com/fsck/k9/ui/messagelist/MessageListAdapter.kt

```diff
@@ -63,11 +63,8 @@ class MessageListAdapter internal constructor(
         alphaFractionAttrId = R.attr.messageListActiveItemBackgroundAlphaFraction,
         backgroundColorAttrId = R.attr.messageListActiveItemBackgroundAlphaBackground,
     )
-    private val selectedItemBackgroundColor: Int = theme.resolveColorAttribute(
```

Comment:
This was the only usage of `Theme.resolveColorAttribute(Int, Int, Int)`. So that can be removed as well.

Reply:
I checked and it's still used in a couple of places.